gel/codegen/cli.py

```diff
@@ -75,6 +74,12 @@ def error(self, message):
     help="Add a mixin to generated dataclasses "
     "to skip Pydantic validation (default is to add the mixin).",
 )
+parser.add_argument(
+    "--allow-user-specified-id",
+    action=argparse.BooleanOptionalAction,
+    default=argparse.SUPPRESS,  # override the builtin help for default
```

Comment:
```suggestion
```

We don't want to suppress the default `False` here as it is a forward binary option.